src/plot/series/canvas-wrapper.js

```diff
@@ -104,8 +104,13 @@ function buildLayers(newChildren, oldChildren) {
   });
 }
 class CanvasWrapper extends Component {
+
   componentDidMount() {
-    this.drawChildren(this.props, null, this.refs.canvas.getContext('2d'));
+    const ctx = this.refs.canvas.getContext('2d');
+    const { pixelRatio } = this.props;
```

Comment:
I think your build is failing bc it expects `const {pixelRatio} = this.props;`

Reply:
Yea, that's the cause. I've modified the README to point out `npm test` doesn't lint the code.